### PR TITLE
Use link to oyeblikksbilde pdf file

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: src/api/utils.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import axios, { AxiosError, AxiosPromise, AxiosResponse } from 'axios';
 import { APP_NAME } from '../util/constants';
 
+export const NAV_INTERN_PREPROD_SUFFIX = 'intern.dev.nav.no';
+export const NAV_INTERN_PROD_SUFFIX = 'intern.nav.no';
+
```
Comment (dragutin-nav) [inline src/api/utils.ts:6]:
```suggestion
```

File: src/page/oyblikksbilde-visning/oyblikksbilde-visning.tsx [hunk 2]
```diff
@@ -25,6 +30,10 @@ function finnOyblikksbilde(
 	return oyblikksbilde ? oyblikksbilde.json : null;
 }
 
+export function handleOnLastNedLenkeClicked(){
+	return null;
+}
+
```
Comment (dragutin-nav) [inline src/page/oyblikksbilde-visning/oyblikksbilde-visning.tsx:36]:
```suggestion
```

**Review by Mathiamu — COMMENTED** [review]
A few small change suggestions 😀

File: src/page/oyblikksbilde-visning/oyblikksbilde-visning.less [hunk 3]
```diff
@@ -37,6 +37,35 @@
...
+				}
+
+				.file_tittel {
+					color: #0067c5;
+					font-weight: bold;
+					font-size: 1rem;
+					line-height: 1rem;
+					cursor: pointer;
+				}
+			}
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply src/page/oyblikksbilde-visning/oyblikksbilde-visning.less:68]: Nice one! Thanks

File: src/page/oyblikksbilde-visning/oyeblikksbilde-egenvurdering.tsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+		<Card className="vedlegg-card">
+			<Systemtittel tag="h2" className="vedlegg-card__header">
+				Svarene dine om behov for veiledning
+			</Systemtittel>
+			<div className="innhold">
+				{data == null && (
+					<>
+						<b>Ingen registrerte data:</b> Personen har ikke registrert svar om behov for veiledning.
+					</>
+				)}
```
Comment (Mathiamu) [inline src/page/oyblikksbilde-visning/oyeblikksbilde-egenvurdering.tsx:70]: This one doesn't seem to show up, at least on the old test users 🤔

Comment (dragutin-nav) [reply src/page/oyblikksbilde-visning/oyeblikksbilde-egenvurdering.tsx:70]: Yep, I saw that...i think i know what is the error

**Review by Mathiamu — APPROVED** [review]
Great job on all the typing and everything! Just a tiny few questions/comments and I think it's good to go 🚀

File: src/page/oyblikksbilde-visning/oyeblikksbilde-registrering.tsx [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+				{data?.profilering && (
+					<>
+						<h3 className="json-key">Profilering</h3>
+						<div className="json-obj">
+							{data.profilering.innsatsgruppe &&
+								visEnkelVerdi('Innsatsgruppe', innsatsgruppeLabel(data.profilering.innsatsgruppe))}
+
+							{data.profilering.alder && visEnkelVerdi('Alder', data.profilering.alder)}
+
+							{data.profilering.jobbetSammenhengendeSeksAvTolvSisteManeder &&
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply src/page/oyblikksbilde-visning/oyeblikksbilde-registrering.tsx:190]: Fixed

File: src/page/oyblikksbilde-visning/oyeblikksbilde-egenvurdering.tsx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+						<span className="json-key">Sist oppdatert: </span>
+						{formatDates(data.sistOppdatert)}
+					</>
+				)}
+				{data?.svar && data?.svar.length > 0 && (
+					<div className="json-array-wrapper">
+						<h3 className="json-key">Svar</h3>
+						<ul className="json-array">
+							{data.svar.map((svar, i) => (
+								<li key={i}>
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply src/page/oyblikksbilde-visning/oyeblikksbilde-egenvurdering.tsx:83]: Fixed :)

File: src/page/oyblikksbilde-visning/dto/CvDto.tsx [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+	sistEndretAvNav: boolean | null;
+	sammendrag: string | null;
+	arbeidserfaring: ArbeidserfaringDtoV2[] | null;
+	utdanning: UtdanningDtoV2[] | null;
+	fagdokumentasjoner: FagdokumentasjonDtoV2[] | null;
+	godkjenninger: GodkjenningDtoV2[] | null;
+	annenErfaring: AnnenErfaringDtoV2[] | null;
+	forerkort: ForerkortDtoV2[] | null;
+	kurs: KursDtoV2[] | null;
+	sertifikater: SertifikatDtoV2[] | null;
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply src/page/oyblikksbilde-visning/dto/CvDto.tsx:13]: removed `sertifikater`

**Pull request merged**